<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initializeUserDeviceDataDialog</class>
 <widget class="QDialog" name="initializeUserDeviceDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Initialize the following data for this User Device:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="userDeviceSettingsCheckBox">
     <property name="text">
      <string>User Device Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="patchNamesCheckBox">
     <property name="text">
      <string>Patch Names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="effectStatesCheckBox">
     <property name="text">
      <string>Effect states</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="FXNamesCheckBox">
     <property name="text">
      <string>FX names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="sceneNamesCheckBox">
     <property name="text">
      <string>Scene names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="colourSetingsCheckBox">
     <property name="text">
      <string>Custom colour settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkAllBox">
     <property name="text">
      <string>Check all</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>initializeUserDeviceDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>initializeUserDeviceDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
